.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -24,7 +24,9 @@ jobs:
       - name: Build with Maven skipping tests
         run: mvn --batch-mode clean verify -DskipTests
       - name: Generate sha256sum files
-        run: find target -maxdepth 1 -name *.jar -exec bash -c 'sha256sum {} > {}.sha256' \;
```

Comment:
Mark this and other generated files as generated in .gitattributes.
Example: https://github.com/exasol/s3-document-files-virtual-schema/blob/main/.gitattributes

Reply:
Done